README.en.md

```diff
@@ -333,6 +333,7 @@ and should be used for performance optimization.
 | [ignore_regex](#ignore_regex)                                                 | all                       | Setting regex expressions on paths that should be excluded from translation |
 | [ignore_class](#ignore_class)                                                 | all                       | Setting the HTML classes that should be excluded from translation           |
 | [no_index_langs](#no_index_langs)                                             | all                       | Setting languages that should not be included in SEO optimization tags      |
```

Comment:
I want to make the difference between `no_index_langs` and `no_hreflang_langs` clear.
Addition to `no_hreflang_langs` behavior, `no_index_langs` ask indexing bot to not index for specified languages.

Reply:
I agree. no_index_langs didn't mention it before either.